lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -304,8 +292,8 @@ function dosomething_northstar_log_request($op, $user, $request_body, $response)
       'user_country' => $user->field_address[LANGUAGE_NONE][0]['country'],
       'fastly_country' => dosomething_settings_get_geo_country_code(),
       'request_values' => json_encode($request_body),
-      'response_code' => $response->code,
-      'response_values' => $response->data,
+      'response_code' => (isset($response['code']) ? $response['code'] : '200'),
```

Comment:
This bit won't work anymore – this was using a Drupal HTTP response object-array-thing which had a `code` property for the status code and a `data` property with the actual response.

We should probably save this when we catch an exception (not sure we do this all that elegantly in Gateway at the moment, so might need a tiny update there), since we want to be able to see what kind of error it is.